Sources/ComposableArchitecture/Documentation.docc/ComposableArchitecture.md

```diff
@@ -60,7 +60,7 @@ day-to-day when building applications, such as:
 ### State management
 
 - ``ReducerProtocol``
-- ``Effect``
+- ``EffectTask``
```

Comment:
```suggestion
- ``EffectPublisher``
```
We're changing also here to redirect on `EffectPublisher`'s page where the interesting documentation lives.